### DISCUSSION TITLE
Mac startup codes

### DISCUSSION THREAD

**Discussion by ttpilot** [BlueSCSI/BlueSCSI-v2#182] [Help]
Does BlueSCSI recognize MacOS startup key combinations? I’ve tried cmd-opt-shift-delete-[SCSI ID] to try booting from different devices on the SD card, but no luck

**Comment by erichelgeson** [comment]
The startup codes are part of the Mac ROM - not BlueSCSI. If there is a bootable device on those SCSI ID's (with a blessed folder) the key combo will work.

**Reply by ttpilot** [reply]
Device 0 on the BlueSCSI is bootable, but at times the system does not seem to see it. This is on a Color Classic Mystic system. The bootable drive, ‘HD0 CC SSD.hda’, usually works. It’s running System 8.1. The other drive folders are not bootable

**Reply by erichelgeson** [reply]
A few things - if it's intermittent it may be a power or termination issue - You should fully go through our [troubleshooting guide](https://github.com/BlueSCSI/BlueSCSI-v2/wiki/Troubleshooting#what-to-try-first) first to rule out any of those common issues.

Second Mystic systems need special enablers and/or hardware mods - not all systems will boot when in a CC. See: https://powercc.org/mystic/ - I personally use the Option 2 software mod with 7.1 and system enabler 065 v1.1.

You can also use/try [System Picker](https://macintoshgarden.org/apps/system-picker) to switch between drives and system folders.
